src/ScottPlot4/ScottPlot/Renderable/Legend.cs

```diff
@@ -98,22 +100,50 @@ public Bitmap GetBitmap(bool lowQuality = false, double scale = 1.0)
             // determine maximum label size and use it to define legend size
             float maxLabelWidth = 0;
             float maxLabelHeight = 0;
+            float totLegendWidth = 0;
```

Comment:
This is confusing, because this variable only holds the total legend width _if_ the legend is horizontal. If it's vertical, it doesn't hold anything of meaning.

Not sure what to rename it to, `legendWidthHorizontal` isn't really my favourite, but it's perhaps an idea.

Reply:
Called `totalLegendWidthWhenHorizontal`